kubernetes-elasticsearch-cluster/es-data-stateful.yaml

```diff
@@ -48,24 +64,36 @@ spec:
         - name: NODE_INGEST
           value: "false"
         - name: HTTP_ENABLE
-          value: "false"
+          value: "true"
         - name: ES_JAVA_OPTS
-          value: -Xms1g -Xmx1g
+          value: -Xms2g -Xmx2g
+        - name: NETWORK_HOST
+          value: _site_,_lo_
         - name: PROCESSORS
           valueFrom:
             resourceFieldRef:
               resource: limits.cpu
         resources:
+          requests:
+            cpu: 2
           limits:
-            cpu: 1
+            cpu: 3
```

Comment:
Remove the limit so it uses what's available?

Reply:
Done (and rm PROCESSORS)